report/Sprint_1/Application.tex

```diff
@@ -0,0 +1,92 @@
+\section{Application}
+During the first sprint, we started development on the application.
+The following features were implemented:
+\begin{itemize}
+    \item Login screen
+    \item Application selection screen
+    \item Create application screen
+    \item Create user screen
+    \item Task dashboard screen
+    \item Create task screen
+\end{itemize}
+
+In this sprint, some of the functionality was mocked, because they were yet to be developed on the backend.
```

Comment:
Synes ikke der er grund til at skrive in this sprint igen, da hele afsnittet omhandler det sprint.
```suggestion
It should be noted that, some of the functionality was mocked, because they were yet to be developed on the backend.
```

Reply:
Jeg har omformuleret det lidt